src/main/java/com/github/onsdigital/thetrain/json/Transaction.java

```diff
@@ -52,19 +61,43 @@ public String id() {
     }
 
     /**
-     * @return The transaction {@link #startDate}.
+     * @return The transaction {@link #startDateObject}.
      */
     public String startDate() {
         return startDate;
     }
 
+    /**
+     * @return The transaction {@link #startDate}.
+     */
+    public Date getStartDateObject() {
+        return startDateObject;
+    }
+
+    /**
+     * @return The transaction {@link #startDate}.
+     */
+    public String getStartDate() {
+        return startDate;
+    }
+    /**
+     * @return The transaction {@link #endDate}.
+     */
+    public String endDate() { return endDate; }
```

Comment:
for consistency with the above statements the return statement should be on the next line